datalad/distribution/get.py

```diff
@@ -273,15 +274,27 @@ def result_renderer_cmdline(res, args):
         if not isinstance(res, list):
             res = [res]
         if not len(res):
-            ui.message("Nothing was getted")
+            ui.message("Got nothing (we have it all already)")
             return
 
-        msg = linesep.join([
-            "{path} ... {suc}".format(
-                suc="ok." if item.get('success', False)
-                    else "failed. (%s)" % item.get('note', 'unknown reason'),
-                path=item.get('file'))
-            for item in res])
+        # provide summary
+        nsuccess = sum(item.get('success', False) for item in res)
+        nfailure = len(res) - nsuccess
+        msg = "Requested %d %s." % (len(res), single_or_plural("file", "files", len(res)))
+        if nsuccess:
+            msg += " Got %d. " % nsuccess
+        if nfailure:
+            msg += " Failed to get %d." % (nfailure,)
         ui.message(msg)
 
+        # if just a few or less than initially explicitly requested
+        if len(res) < 10 or len(res) <= len(args.path):
```

Comment:
`len(res)` is a good idea, I think, but `len(args.path)`?. You could have a generated list as input, that is really huge. I think we should go for an `-v` option.